src/Microsoft.DotNet.VersionTools/Dependencies/MSBuildPackageReferenceUpdater.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.VersionTools.Util;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.VersionTools.Dependencies
+{
+    public class MSBuildPackageReferenceUpdater : IDependencyUpdater
```

Comment:
At first glance I'd like this to share with `ProjectJsonUpdater`, but there isn't much value in that if `ProjectJsonUpdater` goes away like it eventually should.